styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java

```diff
@@ -365,22 +366,29 @@ public void shouldLimitConcurrencyForResource() throws Exception {
 
     assertThat(countInState(State.QUEUED), is(1));
     assertThat(countInState(State.PREPARE), is(3));
+    verify(stats).recordResourceUsed("r1", 3L);
   }
 
   @Test
-  public void shouldCountResourcesOnNonQueuedStates() throws Exception {
+  public void shouldCountResourcesOnStatesNotConsumingResources() throws Exception {
```

Comment:
i think the name of this test is wrong, there's a `not` in there

Reply:
good catch :D